cromshell

```diff
@@ -1447,28 +1527,28 @@ if ${ISINTERACTIVESHELL} ; then
 
   # Validate our sub-command:
   case ${SUB_COMMAND} in
-    cleanup|submit|status|logs|execution-status-count|metadata|slim-metadata|timing|abort|notify|list|fetch-all|fetch-logs|list-outputs)
-      # This is a good sub-command, so we do not need to do anything. 
+    cleanup|submit|status|logs|execution-status-count|pretty-execution-status|metadata|slim-metadata|timing|abort|notify|list|fetch-all|fetch-logs|list-outputs)
```

Comment:
You don't need to add `pretty-execution-status` in here since the desired entry point from the user is still `execution-status-count`.

Reply:
Yup, that was an oversight on my part that I didn't remove after I added the `-p` option.